benchmarks/run_naslib.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+import os
+import subprocess
+
+
+def run(args: argparse.Namespace) -> None:
+    kurobako_cmd = os.path.join(args.path_to_kurobako, "kurobako")
+    subprocess.run(f"{kurobako_cmd} --version", shell=True)
+
+    os.makedirs(args.out_dir, exist_ok=True)
+    study_json_filename = os.path.join(args.out_dir, "studies.json")
+    subprocess.check_call(f"echo >| {study_json_filename}", shell=True)
+    solvers_filename = os.path.join(args.out_dir, "solvers.json")
+    subprocess.check_call(f"echo >| {solvers_filename}", shell=True)
+    problems_filename = os.path.join(args.out_dir, "problems.json")
+    subprocess.check_call(f"echo >| {problems_filename}", shell=True)
```

Comment:
It is good to do the same thing as `run_kurobako.py` to verify the existence of these file.